examples/app-mvvm/app/view/summit/Chart.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * View for chart showing summit heights
+ */
+Ext.define('CF.view.summit.Chart', {
+    extend: 'Ext.panel.Panel',
+    requires: [
+        'Ext.chart.CartesianChart',
+        'Ext.chart.axis.Numeric',
+        'Ext.chart.axis.Category',
+        'Ext.chart.series.Line',
+        'Ext.chart.interactions.ItemHighlight'
+    ],
+
+    xtype : 'cf_summitchart',
+
+    height: 500,
+
+    layout: 'fit',
+
+    // i18n properties
+    bottomAxeTitleText: "Summit",
+    leftAxeTitleText: "Elevation",
```

Comment:
Since series and axis do not inherit from ``Component`` they do not have the ``bind`` property in order to bind the ViewModel's data.